client/src/overview/components/StyleSelector.jsx

```diff
@@ -5,14 +5,12 @@ import Thumbnail from './Thumbnail.jsx';
 let StyleSelector = ({selectedStyle, setSelectedStyle, productStyles}) => {
   if (!selectedStyle || !productStyles) return(<></>)
```

Comment:
@JeromeMTR is returning the empty div for formatting purposes or can you just return null?

Reply:
@crypto-bender I could return null but either way its the same thing. I've returned null on some of them and was still checking if this works